documents/credential-branding.md

```diff
@@ -0,0 +1,91 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>OID4VC demo's credential branding 
+  <br>
+</h1>
+
+---
+
+**Warning: This project still is in very early development. Breaking changes without notice will happen at this point!**
```

Comment:
Remove these warnings from the individual documents in this dir

Reply:
removed all of them in this dir